apple-pie/styles/style.css

```
@@ -21,4 +24,66 @@ body {
 }
 
 /* STYLES */
-/* Write your CSS below */
\ No newline at end of file
+/* Write your CSS below */
+#base{
+  position: relative;
+  background-color: #fff ;
+  width: 1280px;
+  justify-content: center;
+}
+.photo{
+  width: 700px;
+  height: 480px;
+  background-image: url(../images/apple-pie.jpg);
+  background-size: cover;
+  margin-left: auto;
+  margin-right: auto;
+}
```

Recalco, muy muy bien salvado!

```suggestion
.photo{
  display: flex;
  justify-content: center;
  align-items: center;
  width: 700px;
  height: 480px;
  background-image: url(../images/apple-pie.jpg);
  background-size: cover;
  margin-left: auto;
  margin-right: auto;
}
```